.github/workflows/build-and-test-mac-windows.yml

```diff
@@ -1,8 +1,11 @@
 name: Build + Test Mac-Windows
 
 on:
+  pull_request:
```

Comment:
As far as I know we only support Linux now, will this work on windows and mac or it will fail ?

Reply:
It works fine on windows / macos, but not with --race because of CGO things which were introduced with BLS signing (C code / go-binding).
We already had situations where a test on win failed (legitimately) but we didn't saw it on linux.

Without the PR this win test was triggered at the moment a tag was pushed, what doesn't make sense.
Now the win test runs only a little bit earlier...